src/middleware/visualizer.coffee

```diff
@@ -85,9 +85,7 @@ storeVisualizerEvents = (ctx, done) ->
 
   events.VisualizerEvent.create trxEvents, (err) -> return if err then done err else done()
 
-exports.koaMiddleware =  `function *visualizerMiddleware(next) {
-  yield next;
-  if (config.visualizer.enableVisualizer) {
-    storeVisualizerEvents(this, function(){});
-  }
-}`
+exports.koaMiddleware = (next) ->
+  yield next
+  if config.visualizer.enableVisualizer?
```

Comment:
I think you may need to take the question mark out, I suspect that'll just translate to `if (config.visualizer.enableVisualizer != null)`